apps/backend/src/routes/signup/onboard/+page.svelte

```diff
@@ -0,0 +1,87 @@
+<!-- JavaScript code for Continue button -->
+<script lang="ts">
+  let confirm: boolean = false;
+</script>
+
+<!-- Onboard Form - Logo -->
+<div class="skeleton w-32 h-32"></div>
+
+<div class="flex flex-row items-center justify-center h-screen">
+  <!-- Onboard Form - Card -->
+  <div class="w-full max-w-2xl p-10 border border-secondary rounded-md shadow-lg">
+    <!-- Onboard Form - Header -->
+    <h1 class="text-3xl font-semibold text-center mb-5">Get Onboard</h1>
+
+    <h2 class="text-center">Complete your profile to start using Armadillo!</h2>
+
+    <form action="">
+      <!-- Onboard Form - Name Input -->
+      <div>
+        <label class="form-control w-full max-w-xl">
+          <div class="label">
+            <span class="label-text">Full Name</span>
+          </div>
+          <input
+            type="text"
+            name="fullname"
+            placeholder="John Doe"
+            class="input input-bordered w-full max-w-xl
+					focus:ring-secondary focus:border-secondary duration-300 focus:outline-none" />
+        </label>
+      </div>
+
+      <!-- Onboard Form - Corporate Domain Address Input -->
+      <div>
+        <label class="form-control w-full max-w-xl">
+          <div class="label">
+            <span class="label-text">Domain Address</span>
+          </div>
+          <input
+            type="text"
+            name="domainaddress"
+            placeholder="nyp.edu.sg"
+            class="input input-bordered w-full max-w-xl
+					focus:ring-secondary focus:border-secondary duration-300 focus:outline-none" />
+        </label>
+      </div>
+
+      <!-- Onboard Form - Headshot Photo Input -->
+      <div>
+        <label class="form-control w-full max-w-xl">
+          <div class="label">
+            <span class="label-text">Headshot Photo (.png/.jpg)</span>
+          </div>
+          <input
+            type="file"
+            name="headshot"
+            class="file-input file-input-bordered file-input-secondary w-full max-w-xl" />
+        </label>
+      </div>
+
+      <!-- Onboard Form - Submit/Continue Button -->
+      {#if !confirm}
+        <button
+          type="submit"
+          class="btn btn-secondary w-full max-w-xl mt-8 mb-2 hover:ring-2 hover:ring-info"
+          on:click={() => {
+            confirm = true;
+          }}>Create Account</button>
```

Comment:
Keep your `>` on the same line, and other content on another line.

E.g.,
```
<button
  type="submit"
  ...>
    Continue
</button>
```

Reply:
Resolved:

```
<button
  type="submit"
  class="btn btn-secondary w-full max-w-xl mt-8 mb-2 hover:ring-2 hover:ring-info"
  on:click={() => {
    confirm = true;
  }}>
  Create Account
</button>
```